### PR TITLE
refactor: update as per review

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2635

## Why was it changed?
Follow PR

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [ ] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by marcin-kam — APPROVED** [review]

File: R/standardize_MAE.R [hunk 1]
```diff
@@ -334,23 +334,27 @@ set_unique_cl_names <- function(se) {
 #' @export
 #' @keywords standardize_MAE
 set_unique_drug_names <- function(se) {
```
[the remark this answers is not in the document]
Comment (bczech) [reply R/standardize_MAE.R:336]: Gnumbers are unique in REST API so such situation can't exist in our data model

[the remark this answers is not in the document]
Comment (bczech) [reply R/standardize_MAE.R:336]: The former is correct :) The uniqueness results from the drug combinations

[the remark this answers is not in the document]
Comment (j-smola) [reply R/standardize_MAE.R:336]: @bczech Example that explain this 
```
se <- SummarizedExperiment::SummarizedExperiment(
  assays = list(counts = matrix(1:16, ncol = 4)),
  rowData = S4Vectors::DataFrame(DrugName = c("DrugA", "DrugA", "DrugB", "DrugB"),
                                 Gnumber = c("G1", "G2", "G5", "G5"),
                                 DrugName_2 = c("DrugC", "DrugC", "DrugC", "DrugC"),
                                 Gnumber_2 = c("G3", "G3", "G5", "G3")
  ))
se <- set_unique_drug_names(se)
SummarizedExperiment::rowData(se)
```

**Review by j-smola — APPROVED** [review]

File: R/standardize_MAE.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (bczech) [reply R/standardize_MAE.R:336]: updated :D

[the remark this answers is not in the document]
Comment (bczech) [reply R/standardize_MAE.R:336]: so then we also receive:

```
DataFrame with 3 rows and 4 columns
     DrugName     Gnumber  DrugName_2   Gnumber_2
  <character> <character> <character> <character>
1  DrugA (G1)          G1   DrugC (G3)         G3
2  DrugA (G2)          G2   DrugC (G3)         G3
3       DrugB          G5   DrugC (G5)         G5
```
for the first example that is also correct (unique)

[the remark this answers is not in the document]
Comment (j-smola) [reply R/standardize_MAE.R:336]: Nice! :) 

```
DataFrame with 4 rows and 4 columns
     DrugName     Gnumber  DrugName_2   Gnumber_2
  <character> <character> <character> <character>
1  DrugA (G1)          G1  DrugC (G3)          G3
2  DrugA (G2)          G2  DrugC (G3)          G3
3       DrugB          G5  DrugC (G5)          G5
4       DrugB          G5  DrugC (G3)          G3
```

**Pull request merged**